compiler/native/validate.go

```diff
@@ -16,7 +16,7 @@ func (c *client) Validate(p *yaml.Build) error {
 	var result error
 	// check a version is provided
 	if len(p.Version) == 0 {
-		result = multierror.Append(result, fmt.Errorf("no \"version:\" YAML property provided"))
```

Comment:
I think we want to keep this. Any reason you removed it?

Reply:
Good catch, I think I did a repo wide replacement of that string and accidently changed that.